feature/intern/src/main/java/com/terning/feature/intern/InternRoute.kt

```diff
@@ -64,46 +67,61 @@ fun InternRoute(
         viewModel.sideEffect.flowWithLifecycle(lifecycle = lifecycleOwner.lifecycle)
             .collect { sideEffect ->
                 when (sideEffect) {
-                    is InternViewSideEffect.Toast -> context.toast(sideEffect.message)
+                    is InternViewSideEffect.ShowToast -> context.toast(sideEffect.message)
                 }
             }
     }
 
-    when (internState.loadState) {
-        UiState.Loading -> {}
-        UiState.Empty -> {}
-        is UiState.Failure -> {}
-        is UiState.Success -> {
-            InternScreen(
-                announcementId = announcementId,
-                internUiState = internState,
-                internInfo = (internState.loadState as UiState.Success).data,
-                navController = navController,
-                onDismissCancelDialog = {
-                    with(viewModel) {
-                        updateScrapCancelDialogVisibility(false)
-                        getInternInfo(announcementId)
-                    }
-                },
-                onDismissScrapDialog = {
-                    with(viewModel) {
-                        updateInternDialogVisibility(false)
-                        getInternInfo(announcementId)
-                    }
-                },
-                onClickCancelButton = {
-                    viewModel.updateScrapCancelDialogVisibility(true)
-                },
-                onClickScrapButton = {
-                    amplitudeTracker.track(
-                        type = EventType.CLICK,
-                        name = "detail_scrap"
-                    )
-                    viewModel.updateInternDialogVisibility(true)
-                }
+
+    val internInfo = when (internState.loadState) {
+        is UiState.Success -> (internState.loadState as UiState.Success<InternInfo>).data
+        else -> InternInfo(
+            dDay = "",
+            title = "",
+            viewCount = 0,
+            company = "",
+            companyCategory = "",
+            companyImage = "",
+            deadline = "",
+            workingPeriod = "",
+            startYearMonth = "",
+            qualification = "",
+            jobType = "",
+            url = "",
+            detail = "",
+            isScrapped = false,
+            scrapCount = 0
+        )
```

Comment:
```suggestion
    val internInfo = (internState.loadState as? UiState.Success<InternInfo>)?.data ?: InternInfo()
```

석준오빠가 리뷰 단 것처럼 빈 InternInfo 반환할 수 있도록 만들면 이렇게 구현하는 것도 깔끔하고 좋을 것 같아요!